src/xpcc/architecture/platform/devices/stm32/stm32f072-c_r_v-8_b.xml

```diff
@@ -51,7 +51,7 @@
       <vector position="30" name="CEC_CAN"/>
       <vector position="31" name="USB"/>
     </driver>
-    <driver type="adc" name="stm32f0"/>
+<!--     <driver type="adc" name="stm32f0"/> -->
```

Comment:
well here the question is: do the device files describe the device or do they describe the build process for `xpcc`?